plugins/release.py

```diff
@@ -0,0 +1,161 @@
+                for system, status in statuses.items():
+                    # If only want to see paused systems -and- the system is active, skip.
+                    if paused_only and not len(status):
+                        continue
+                    if len(status):
+                        status_str = "PAUSED for {} reason(s)".format(len(status))
+                    else:
+                        status_str = "ACTIVE"
+                    cmd_output += "{:>10}: {}\n".format(system, status_str)
+            else:
+                # No statuses - must mean that no paused pipeline systems exist.
+                cmd_output += "     All systems ({}) active.".format(', '.join(PIPELINE_SYSTEM_INFO.keys()))
+        return cmd_output
+
+    @respond_to(r"^pause[\s]+"
+                r"(?P<pipeline_system>\w*)[\s]+"  # Pipeline system to pause
+                r"because[\s]+"
+                r"(?P<pause_reason>.*)")  # Reason for pausing
+    def pause(self, message, pipeline_system, pause_reason):
+        """
+        pause [pipeline_system] because [reason]
+            : Pause the specified pipeline system for the specified reason.
+        """
+        if not self._check_pipeline_system(pipeline_system):
+            return
+
+        pause_status = self.pause_ops.add_pipeline_event(message.sender.nick, pipeline_system, pause_reason)
+        response = "Added pause event {} for system '{}'".format(
+            pause_status['event_id'],
+            pipeline_system
+        )
+        if pause_status['paused']:
+            response += " - paused the system."
+        else:
+            response += ", which is already paused."
+        self._say(response)
+
+    @respond_to(r"^event[\s]+"
+                r"(?P<event_id>\w*)[\s]+"  # Pipeline event to remove
+                r"resolved")
+    def remove_event(self, message, event_id):
+        """
+        event [event_id] resolved
+            : Remove the specified pipeline pause event.
+        """
+        try:
+            remove_status = self.pause_ops.remove_pipeline_event(message.sender.nick, event_id)
+        except PauseEventNotFound:
+            self._say_error("Event '{}' was not found.".format(event_id))
+        except MultiplePauseEventsFound:
+            self._say_error(
+                "Multiple events found with ID '{}'? Should not happen - check the S3 bucket.".format(event_id)
+            )
+        else:
+            response = "Event '{}' successfully removed from system '{}'".format(
+                event_id,
+                remove_status['pipeline_system']
+            )
+            if remove_status['unpaused']:
+                response += " - unpaused the system."
+            else:
+                response += " - {} pause event(s) remaining.".format(remove_status['num_remaining_events'])
+            self._say(response)
+
+    @respond_to(r"^(?P<paused_or_not>paused[\s]+pipeline|pipeline)[\s]+"  # Whether paused pipeline filter is on.
```

Comment:
No - this match either `paused pipeline` or `pipeline` at the beginning of the `[paused] pipeline status [pipeline_system]` command. Are you suggesting a change to the Alton command wording?